### ISSUE TITLE
'Add to Portfolio' language should respect group type labels

### ISSUE THREAD

**Issue by boonebgorges** [cuny-academic-commons/commons-in-a-box#311]
If you change 'Portfolio' to something else in CBOX, it's not reflected in the user-facing strings in openlab-portfolio. 

I don't think we want openlab-portfolio to depend on CBOX, so we can't directly use `$group_type->get_label()`. Probably the only solution is to wrap the strings in `apply_filters()` in openlab-portfolio, and then filter them in `cbox-openlab-core`.

**Comment by boonebgorges** [comment]
I've made the necessary changes. To summarize:

- I created a function `openlab_portfolio_get_label()` in openlab-portfolio, which passes the translated string through a filter. It only supports `add_to_portfolio` and `added_to_my_portfolio`. There's currently no support for the admin-facing strings.
- I changed all references to `Add to Portfolio` and `Added to my Portfolio` in `views` to use this new helper function. I also made a change so that a hardcoded string in the JS file is now translatable.
- In cbox-openlab-core, I added support for new labels, `add_to_portfolio` and `added_to_my_portfolio`, linked to the Portfolio group type. I also wrote an Upgrade class to add these strings to a new installation.
- I added a filter callback to swap the openlab-portfolio strings with those defined in CBOX-OL
